SPECS/go-md2man/go-md2man.spec

```diff
@@ -32,28 +32,33 @@ export GOCACHE=%{OUR_GOPATH}/.cache
 export CGO_ENABLED=0
 export GO111MODULE=on
 
-cd %{_topdir}/BUILD/%{name}-%{version}/go-md2man-2.0.0
+cd %{_topdir}/BUILD/%{name}-%{version}/go-md2man-%{version}
 go build -mod vendor -o go-md2man
 
 %install
 mkdir -p "%{buildroot}%{_bindir}"
-cp -aT go-md2man-2.0.0/go-md2man %{buildroot}%{_bindir}/go-md2man
+cp -aT go-md2man-%{version}/go-md2man %{buildroot}%{_bindir}/go-md2man
 
 # copy legal files
 mkdir -p %{buildroot}%{_docdir}/%{name}-%{version}
-cp go-md2man-2.0.0/LICENSE.md %{buildroot}%{_docdir}/%{name}-%{version}/LICENSE.md
+cp go-md2man-%{version}/LICENSE.md %{buildroot}%{_docdir}/%{name}-%{version}/LICENSE.md
 
 %files
 %license %{_docdir}/%{name}-%{version}/LICENSE.md
 %{_bindir}/go-md2man
 
 %changelog
-* Fri Jun 18 2021 Henry Li <lihl@microsoft.com> - 2.0.0-7
+* Fri Mar 04 2022 Andrew Phelps <anphel@microsoft.com> 2.0.1-1
```

Comment:
```suggestion
* Fri Mar 04 2022 Andrew Phelps <anphel@microsoft.com> - 2.0.1-1
```

Reply:
Keeping this as-is only because it matches all of the existing changelog entries